membership_file/admin.py

```diff
@@ -1,11 +1,62 @@
 from django.contrib import admin
 from .models import Member, MemberLog, MemberLogField
-from django.contrib import admin
+from django.utils.html import format_html
+
+
+class HideRelatedNameAdmin(admin.ModelAdmin):
```

Comment:
why this change instead of overriding the string representation?

Reply:
It'd make debugging a lot trickier; printing/loggin the instance would then show nothing.
A non-hacky solution would be to modify tha admin template, but that seemed overkill.